helm/charts/hpe-csi-driver/README.md

```diff
@@ -36,8 +36,8 @@ The following table lists the configurable parameters of the HPE-CSI chart and t
 | storageClass.parameters.fsType                    | Type of file system being used (ext4, ext3, xfs, btrfs)     | xfs         |
 | storageClass.parameters.volumeDescription         | Volume description for volumes created using HPE CSI driver     | -         |
 | storageClass.parameters.accessProtocol            | Access protocol to use for storage connectivity (iscsi, fc)     | iscsi         |
-| storageClass.parameters.provisioningType         | Provisioning type for HPE 3PAR Primera     | tpvv         |
-| storageClass.parameters.cpg            | CPG type for HPE 3PAR Primera     | FC_r6         |
+| storageClass.parameters.provisioningType         | Provisioning type for HPE 3PAR Primera(tpvv, tdvv, full)     | tpvv         |
+| storageClass.parameters.cpg            |  Existing CPG name for HPE 3PAR Primera     | FC_r6         |
```

Comment:
CSP specific `StorageClass` parameters should not leak out into generic parts. Please remove.

Reply:
Removed